tests/catalog/test_sql.py

```diff
@@ -225,6 +237,93 @@ def test_creation_from_impl(catalog_name: str, warehouse: Path) -> None:
     )
 
 
+def confirm_no_tables_exist(alchemy_engine: Engine) -> list[str]:
+    inspector = inspect(alchemy_engine)
+    for c in SqlCatalogBaseTable.__subclasses__():
+        if inspector.has_table(c.__tablename__):
+            c.__table__.drop(alchemy_engine)
+
+    catalog_tables = [c.__tablename__ for c in SqlCatalogBaseTable.__subclasses__()]
```

Comment:
This function seems to serve two purposes:
1. confirm that no tables exist
2. return the list of `SqlCatalogBaseTable.__subclasses__()`

(2) is a static list of tables required for the SqlCatalog. Would it be cleaner to separate out (2) to a global variable in this module, instead of returning it and passing it as an input to `confirm_all_tables_exist` ?